auto_nag/team_managers.py

```diff
@@ -36,3 +40,81 @@ def get_team_manager(
                 return None
 
         return self.managers[team_name]
+
+    def _fetch_component_teams(self):
+        include_fields = [
+            "components.name",
+            "components.team_name",
+        ]
+
+        def handler(product, data):
+            data.update(
+                {
+                    component["name"]: component["team_name"]
+                    for component in product["components"]
+                }
+            )
+
+        # This is workaround until merging https://github.com/mozilla/libmozdata/pull/198
+        search = "type=accessible&include_fields=" + ",".join(include_fields)
+        BugzillaProduct(
+            search_strings=search,
+            product_handler=handler,
+            product_data=self.component_teams,
+        ).wait()
+
+    def _fetch_managers_nicnames(self):
```

Comment:
```suggestion
    def _fetch_managers_nicknames(self):
```

Reply:
Fixed in 3a07f354bb1d72fe591ada0aabd01215ca7655fc